src/commands/eval.js

```diff
@@ -146,12 +146,14 @@ class EvalCommand extends FaunaCommand {
     }
   }
 
-  // Flags should be
-  // {
-  //   version: "4" | "10";
-  //   format: "json" | "json-tagged" | "shell";
-  //   typecheck?: boolean;
-  // }
+  /**
```

Comment:
😂 My bad, I meant to use JS docs like with `@param` lines. I can take a stab at it since this is my nitpick.

Reply:
I dislike `@param` lines because they make it difficult to read. This is just my dislike of javadoc syntax, but I'd prefer to just use words in comments is all.

If you feel strongly about it feel free to go add the `@param` bit, I just think it makes things harder to read.

Reply:
makes sense I guess. I don't use those docs lol, I just rely on typescript signatures.

Reply:
Since this is JS and not TS, there is no TS signature.  The JS docs can actually give you a TS signature.

current signature preview
![image](https://github.com/fauna/fauna-shell/assets/7998974/6702968e-7052-4cdd-987e-c5ddfd27d3f2)

updated signature preview
![image](https://github.com/fauna/fauna-shell/assets/7998974/103e7b79-112b-4a19-8441-7facea24e936)